capellambse/aird/parser/_filters/global.py

```diff
@@ -12,6 +12,40 @@
 from . import FilterArguments, composite, global_filter
 
 XT_CEX_FEX_ALLOCATION = "org.polarsys.capella.core.data.fa:ComponentExchangeFunctionalExchangeAllocation"
+EXCHANGES_WITH_ROLES = frozenset(
+    ("Association", "Aggregation", "Composition", "Generalization")
```

Comment:
```suggestion
    {"Aggregation", "Association", "Composition", "Generalization"}
```

(This matches the `repr` of frozensets)